app/models/form/setup/questions/scheme_id.rb

```diff
@@ -21,7 +21,7 @@ def answer_options
 
   def displayed_answer_options(case_log)
     organisation = case_log.owning_organisation || case_log.created_by&.organisation
-    schemes = organisation ? Scheme.select(:id).where(owning_organisation_id: organisation.id) : Scheme.select(:id)
+    schemes = organisation ? Scheme.select(:id).where(owning_organisation_id: organisation.id).where(confirmed: true) : Scheme.select(:id).where(confirmed: true)
```

Comment:
Not important but any reason not to just make this 1 where query? (Don't think we need `where().where()` right?